### PR TITLE
chore: update proxy-container README

### PR DESCRIPTION
**Pull request by didinele**
**Please describe the changes this PR makes and why it should be merged:**

**Status and versioning classification:**

### CODE REVIEW

File: packages/proxy-container/README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
   <br />
   <p>
     <a href="https://discord.gg/djs"><img src="https://img.shields.io/discord/222078108977594368?color=5865F2&logo=discord&logoColor=white" alt="Discord server" /></a>
-    <a href="https://www.npmjs.com/package/@discordjs/proxy"><img src="https://img.shields.io/npm/v/@discordjs/proxy.svg?maxAge=3600" alt="npm version" /></a>
-    <a href="https://www.npmjs.com/package/@discordjs/proxy"><img src="https://img.shields.io/npm/dt/@discordjs/proxy.svg?maxAge=3600" alt="npm downloads" /></a>
+    <a href="https://hub.docker.com/r/discordjs/proxy"><img src="https://img.shields.io/docker/v/discordjs/proxy.svg?sort=semver&maxAge=3600" alt="dockerhub version" /></a>
+    <a href="https://hub.docker.com/r/discordjs/proxy"><img src="https://img.shields.io/docker/pulls/discordjs/proxy?maxAge=3600" alt="dockerhub pulls" /></a>
```
Comment (almeidx) [inline packages/proxy-container/README.md:10]: Any reason as to why the `.svg` was removed from the second one 🧐

Comment (didinele) [reply packages/proxy-container/README.md]: ty

File: packages/proxy-container/README.md [hunk 2]
```diff
@@ -6,8 +6,8 @@
   <br />
   <p>
     <a href="https://discord.gg/djs"><img src="https://img.shields.io/discord/222078108977594368?color=5865F2&logo=discord&logoColor=white" alt="Discord server" /></a>
-    <a href="https://www.npmjs.com/package/@discordjs/proxy"><img src="https://img.shields.io/npm/v/@discordjs/proxy.svg?maxAge=3600" alt="npm version" /></a>
-    <a href="https://www.npmjs.com/package/@discordjs/proxy"><img src="https://img.shields.io/npm/dt/@discordjs/proxy.svg?maxAge=3600" alt="npm downloads" /></a>
+    <a href="https://hub.docker.com/r/discordjs/proxy"><img src="https://img.shields.io/docker/v/discordjs/proxy.svg?sort=semver&maxAge=3600" alt="dockerhub version" /></a>
+    <a href="https://hub.docker.com/r/discordjs/proxy"><img src="https://img.shields.io/docker/pulls/discordjs/proxy?maxAge=3600.svg" alt="dockerhub pulls" /></a>
```
Comment (almeidx) [inline packages/proxy-container/README.md:10]:
```suggestion
    <a href="https://hub.docker.com/r/discordjs/proxy"><img src="https://img.shields.io/docker/pulls/discordjs/proxy.svg?maxAge=3600" alt="dockerhub pulls" /></a>
```

Comment (didinele) [reply packages/proxy-container/README.md:10]: god damn it

**Comment by didinele** [comment]
just realized I should also change npm->dockerhub in the links section. will do that once I'm home

**Review by kyranet — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Pull request merged**